www/caddy/README.md

```diff
@@ -0,0 +1,205 @@
+# Caddy Plugin for OPNsense
```

Comment:
just as a side node this fits rather well for a page in the docs repo instead of dangling here in the repo but it's not a problem. just something for the wishlist.

Reply:
Yes my plan is to write a doc page too eventually.